docs/content/preview/migrate/yb-voyager-cli.md

```diff
@@ -391,7 +401,7 @@ The following table summarizes the arguments and options you can pass to yb-voya
 | | `--source-ssl-cert` <br /> `--source-ssl-key` | These two arguments specify names of the files containing SSL certificate and key, respectively. The `<cert, key>` pair forms the identity of the client. |
 | | `--source-ssl-root-cert` | Specifies the path to a file containing SSL certificate authority (CA) certificate(s). If the file exists, the server's certificate will be verified to be signed by one of these authorities.
 | | `--source-ssl-crl` | Specifies the path to a file containing the SSL certificate revocation list (CRL). Certificates listed in this file, if it exists, will be rejected while attempting to authenticate the server's certificate.
-| Oracle | `--oracle-tns-alias` | A TNS alias that is configured to establish a secure connection with the server is passed to yb-voyager. When you pass this argument, you don't need to pass the `--source-db-host`, `--source-db-port`, and `--source-db-name` arguments to yb-voyager.|
+| Oracle | `--oracle-tns-alias` | A TNS alias that is configured to establish a secure connection with the server is passed to yb-voyager. When you pass this argument, you cannot use any other arguments to connect to your Oracle instance. To connect to Oracle instances you can use only one from the following arguments: <br/> [`--source-db-schema`](../yb-voyager-cli/#source-db-schema) <br/> [`--oracle-db-sid`](../yb-voyager-cli/#oracle-db-sid)<br/>[`--oracle-tns-alias`](../yb-voyager-cli/##ssl-connectivity)|
```

Comment:
The 3 argument choices out of which one is compulsory are source-db-name, oracle-tns-alias and oracle-db-sid. There are other mandatory arguments that have to be provided irrespective of source-db-type, so please also omit the fragment `you can use only one from the following arguments` It is true that the oracle-tns-alias usually has all the required arguments needed to connect to the source db, but it is not necessarily true while using the other two modes of connection (source-db-name, oracle-db-sid)